app/scripts/kunden/detail/kundendetail.html

```diff
@@ -405,7 +427,7 @@ <h3 class="panel-title">
                 {{rechnung.status | translate}}
               </td>
               <td>
-                {{rechnung.eingangsDatum | translate}}
+                {{rechnung.eingangsDatum | date}}
```

Comment:
why date and not translate?

Reply:
Because a date "Datum" needs to be formated i.e. DD.MM.YYYY and not be translated